packages/renderer/src/lib/deployments/DeploymentsList.svelte

```diff
@@ -126,7 +127,14 @@ const row = new TableRow<DeploymentUI>({ selectable: _deployment => true });
   <svelte:fragment slot="bottom-additional-actions">
     {#if selectedItemsNumber > 0}
       <Button
-        on:click="{() => deleteSelectedDeployments()}"
+        on:click="{() =>
+          window
+            .getConfigurationValue('userConfirmation.bulk')
+            .then(value =>
+              value
+                ? withConfirmation(deleteSelectedDeployments, `delete ${selectedItemsNumber} deployments`)
+                : deleteSelectedDeployments(),
+            )}"
```

Comment:
Maybe having an utility function somewhere similar to withConfirmation

```ts
export async function withBulkConfirmation(callback: () => void, text: string): Promise<void> {
  const confirm: boolean = await window.getConfigurationValue('userConfirmation.bulk');
  if(confirm) return withConfirmation(callback, text);
  else callback();
}
```

Reply:
ℹ️ On github don't hesite to respond/or `Resolve` the comments once they are addressed!